styles/standard.mapcss

```diff
@@ -816,10 +816,11 @@ node|z14-[railway=tram_stop]
 /*****************/
 /* owner changes */
 /*****************/
-node|z11-[railway=owner_change]
+node|z11-[railway=border],
```

Comment:
Sorry, I did not read that. It is not the best solution to use the same icon for multiple features, but at the moment it is ok as a quick solution. We can think of an alternative icon in the future.